app/views/expenses/_expense.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="container card mb-4">
+	<div class="p-4">
+		<div class="d-flex justify-content-between flex-column">
+			<h5><%= expense.name %></h5>
+			<small>$<%= expense.amount.round(2) %></small>
+			<%= button_to "Delete", expense_path(expense), :method => :delete, :class => "btn btn-danger btn-sm"%>
+		</div>
+		<small class='text-muted'><%= expense.created_at.strftime("%d %b %Y") %></small>
+	</div>
+</div>
```

Comment:
- [ ] (I added the time myself for illustration purposes) You are required to present your transactions (expenses) in order of the most recent one. As you can see from the time, the order should be: `Xavineta`, `Messi hat`, `Messiverse`. Kindly set up your view accordingly.

  ![ordered-recent](https://user-images.githubusercontent.com/36542986/164035939-3fe9c214-2ba3-4980-8139-4439bea33e32.png)

Reply:
Done